ONSdigital/dp-frontend-router
# add mystudy survey redirect

### What

Add a redirect for the mystudy survey

### How to review

* Check redirect from /mystudy to https://surveys.ons.gov.uk/mystudy
* Run frontend router and test that redirect works

### Who can review

Anyone except @ian-kent
